backend/index.js

```diff
@@ -7,6 +7,13 @@ const prisma = new PrismaClient()
 const session = require('express-session');
 const cors = require('cors');
 const userRoutes = require('./users.js');
+const audioRoutes = require('./audios.js');
+
+
+const bodyParser = require('body-parser'); 
+
+app.use(bodyParser.json({ limit: '50mb' }));
+app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }));
```

Comment:
Does this allow you to process the audio data in the request if the data is large?